Scientific_workflows/FractionalCoverOfWaterProject/Manual_classification_workflow.ipynb

```diff
@@ -0,0 +1,994 @@
+{
```

Comment:
<p>Maybe this should be in supplementary data?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/663/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>It depends on where the user will upload or save the drone imagery. No need for a compulsory dir.</p>